coralproject/experiment-playground
# Block User

## What does this PR do?

Adds functionality to block a user.
## How do I test this PR?
- Open the moderation toggle group
- Activate the “Block Toggle”
- It should show a menu and allow you to block a user.
